File: src/main/java/lengthOfLastWord/LengthOfLastWord.java
/*
 * @Source: https://leetcode-cn.com/problems/
 * @Author: Tian Zhuo
 * @Date: 2018-12-26 22:17:07
 */
package lengthOfLastWord;

public class LengthOfLastWord {
    public int lengthOfLastWord(String s) {
        int count = 0;
        char[] cs = s.toCharArray();
        for (int i = cs.length - 1; i >= 0; i--) {
            if (cs[i] != ' ') {
                while (i > -1 && cs[i] != ' ') {
                    count++;
                    i--;
                }
                break;
            }
        }
        return count;

    }
}

File: algorithms/js/lengthOfLastWord/lengthOfLastWord.js
/*
 * @Source: https://leetcode-cn.com/problems/length-of-last-word/
 * @Author: Tian Zhuo
 * @Date: 2018-12-30 22:20:17
 */
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    s = s.trim();
    return s.length - s.lastIndexOf(' ') - 1;
};